how-to/releaseduty/merge-duty/merge_duty.rst

```diff
@@ -235,8 +235,8 @@ Merge central to beta
    The diff should look like `this
    <https://hg.mozilla.org/releases/mozilla-beta/rev/5f344535f8a3340fa51528be88e7104538b64b2e>`__
 3. At the same time ``mozilla-central`` should get a new tag
-   `in the form FIREFOX_BETA_xxx_BASE <https://hg.mozilla.org/mozilla-central/rev/592c2df16ac45a09c837b8a281e366c419c8b94d>`__
-   - where xxx is also the major Gecko version that mozilla-central had prior to the version bump.
+   `in the form FIREFOX_BETA_yyy_BASE <https://hg.mozilla.org/releases/mozilla-beta/rev/592c2df16ac45a09c837b8a281e366c419c8b94d>`__
```

Comment:
Seems best to keep this link pointing at the mozilla-central version of the revision.

Reply:
Oh, woops.